### PR TITLE
Fix broken link. Fix broken formatting in code block

### PR DESCRIPTION
**Pull request by Pebaz**
Fix broken link. Fix broken formatting in code block. Add links to HLSL Semantics docs.

### CODE REVIEW

**Comment by PRMerger9** [comment]
@Pebaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/d3d12/ns-d3d12-d3d12_rasterizer_desc.md [hunk 1]
```diff
@@ -81,18 +81,18 @@ Scalar on a given pixel's slope. For info about depth bias, see <a href="/window
...
 ``` syntax
 
-0 &lt; w
--w &lt;= x &lt;= w (or arbitrarily wider range if implementation uses a guard band to reduce clipping burden)
--w &lt;= y &lt;= w (or arbitrarily wider range if implementation uses a guard band to reduce clipping burden)
-0 &lt;= z &lt;= w
+0 < w
+-w <= x <= w (or arbitrarily wider range if implementation uses a guard band to reduce clipping burden)
+-w <= y <= w (or arbitrarily wider range if implementation uses a guard band to reduce clipping burden)
+0 <= z <= w
```
Comment (Pebaz) [inline sdk-api-src/content/d3d12/ns-d3d12-d3d12_rasterizer_desc.md:95]: Here's the first non-whitespace code change.

File: sdk-api-src/content/d3d12/ns-d3d12-d3d12_shader_bytecode.md [hunk 2]
```diff
@@ -70,8 +70,6 @@ When loading a shader from FXC/DXC, this may be the entire compiled blob as is l
 
 ## -see-also
 
-<a href="/windows/desktop/direct3d12/commandlistcast">CD3DX12_SHADER_BYTECODE</a>
-
-
+<a href="/windows/desktop/direct3d12/cd3dx12-shader-bytecode">CD3DX12_SHADER_BYTECODE</a>
```
Comment (Pebaz) [inline sdk-api-src/content/d3d12/ns-d3d12-d3d12_shader_bytecode.md:73]: Here's the second non-whitespace code change.

**Comment by PRMerger14** [comment]
@Pebaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/d3d12/ns-d3d12-d3d12_input_element_desc.md [hunk 3]
```diff
@@ -56,12 +56,12 @@ Describes a single element for the input-assembler stage of the graphics pipelin
 
 ### -field SemanticName
 
-The HLSL semantic associated with this element in a shader input-signature.
+The HLSL semantic associated with this element in a shader input-signature. See <a href="/windows/win32/direct3dhlsl/dx-graphics-hlsl-semantics">HLSL Semantics</a> for more info.
```
Comment (Pebaz) [inline sdk-api-src/content/d3d12/ns-d3d12-d3d12_input_element_desc.md:59]: Non-whitespace.

File: sdk-api-src/content/d3d12/ns-d3d12-d3d12_input_element_desc.md [hunk 4]
```diff
@@ -92,4 +92,5 @@ This structure is a member of the <a href="/windows/desktop/api/d3d12/ns-d3d12-d
 
 ## -see-also
 
-<a href="/windows/desktop/direct3d12/direct3d-12-structures">Core Structures</a>
\ No newline at end of file
+<a href="/windows/desktop/direct3d12/direct3d-12-structures">Core Structures</a>
+<a href="/windows/win32/direct3dhlsl/dx-graphics-hlsl-semantics">HLSL Semantics</a>
```
Comment (Pebaz) [inline sdk-api-src/content/d3d12/ns-d3d12-d3d12_input_element_desc.md]: Non-whitespace.

File: sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md [hunk 5]
```diff
@@ -58,15 +58,15 @@ A description of a single element for the input-assembler stage.
 
 Type: <b><a href="/windows/desktop/WinProg/windows-data-types">LPCSTR</a></b>
 
-The HLSL semantic associated with this element in a shader input-signature.
+The HLSL semantic associated with this element in a shader input-signature. See <a href="/windows/win32/direct3dhlsl/dx-graphics-hlsl-semantics">HLSL Semantics</a> for more info.
```
Comment (Pebaz) [inline sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md:61]: Non-whitespace.

**Comment by PRMerger12** [comment]
@Pebaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md (hunk 5 above)
Comment (stevewhims) [reply sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md:61]: FYI, markdown is preferred over HTML.

**Review by stevewhims — APPROVED** [review]

File: sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md (hunk 5 above)
Comment (Pebaz) [reply sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md:61]: Thanks for giving me a heads up, I'll use pure Markdown going forward 🙂

**Pull request merged**